packages/ui/src/Provider/Provider.tsx

```diff
@@ -4,7 +4,7 @@ import merge from 'lodash/merge';
 
 import defaultTheme from './theme/defaultTheme';
 import { ThemeInterface } from './theme/theme.types';
-import Context from './Context';
+import Context, { defaultContextValue } from './Context';
```

Comment:
defaultContextValue why do we need this?

Reply:
to provide the 'getElement' function from the default context

Reply:
Please move it to provider